src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -292,91 +233,63 @@ class AdditionalDetailsStep extends React.Component {
-                                    }}
-                                    errors={{
-                                        street: this.getErrors().addressStreet,
-                                        state: this.getErrors().addressState,
-                                        city: this.getErrors().addressCity,
-                                        zipCode: this.getErrors().addressZip,
-                                    }}
-                                    onFieldChange={(values) => {
-                                        const renamedFields = {
-                                            street: 'addressStreet',
-                                            state: 'addressState',
-                                            city: 'addressCity',
-                                            zipCode: 'addressZip',
-                                        };
-                                        _.each(values, (value, inputKey) => {
-                                            const renamedInputKey = lodashGet(renamedFields, inputKey, inputKey);
-                                            this.clearErrorAndSetValue(renamedInputKey, value);
-                                        });
-                                    }}
+                                    shouldSaveDraft
                                 />
                             </View>
                             <TextInput
+                                inputID={INPUT_IDS.PHONE_NUMBER}
                                 containerStyles={[styles.mt4]}
                                 keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
                                 label={this.props.translate(this.fieldNameTranslationKeys.phoneNumber)}
-                                onChangeText={val => this.clearErrorAndSetValue('phoneNumber', val)}
-                                value={this.props.walletAdditionalDetailsDraft.phoneNumber || ''}
+                                defaultValue={this.getPhoneNumber()}
                                 placeholder={this.props.translate('common.phoneNumberPlaceholder')}
-                                errorText={this.getErrorText('phoneNumber')}
+                                shouldSaveDraft
                             />
                             <DatePicker
+                                inputID={INPUT_IDS.DOB}
                                 containerStyles={[styles.mt4]}
                                 label={this.props.translate(this.fieldNameTranslationKeys.dob)}
-                                onInputChange={val => this.clearDateErrorsAndSetValue(val)}
-                                defaultValue={this.props.walletAdditionalDetailsDraft.dob || ''}
                                 placeholder={this.props.translate('common.dob')}
-                                errorText={this.getErrorText('dob') || this.getErrorText('age')}
+                                maximumDate={new Date()}
```

Comment:
Not sure where this `maximumDate` prop came from. I don't see it referenced anywhere else, so I don't think it's doing anything here.